budget/app.py

```diff
@@ -116,35 +232,41 @@ def create_budgets(user_ids_without_budget, teams_by_user_id):
   for synapse_id in user_ids_without_budget:
     team = teams_by_user_id[synapse_id][0]
     create_budget(synapse_id, team)
+    create_budget_notifications(synapse_id, team)
```

Comment:
you didn't mention in your commit message but i assume you will need to remove the notification as well when a user is removed from a team?

next line: is there a way to confirm budget was successfully created before adding to new_budgets_created list?

Reply:
No, when you remove the budget it wipes out the notifications.

If the response is `{}` and not an error, then the budget was created.

Reply:
i see